augur/application/db/models/augur_data.py

```diff
@@ -3600,4 +3600,50 @@ class RepoClone(Base):
     count_clones = Column(BigInteger)
     clone_data_timestamp = Column(TIMESTAMP(precision=6))
 
-    repo = relationship("Repo")
\ No newline at end of file
+    repo = relationship("Repo")
+
+class ContributorEngagement(Base):
+    __tablename__ = "contributor_engagement"
```

Comment:
@ABrain7710 / @Ulincsys : Can you confirm that its our practice not to modify the main script for table creation, but to have the versioning script also included so that new installs just get "all the upgrades"?